### PR TITLE
sqlmap: add page

### PR DESCRIPTION
**Pull request by hbostann**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by agnivade** [comment]
@hbostann - In future, you can update the same PR by pushing a new commit. You don't have to close and create a new PR every time. :)

File: pages/common/sqlmap.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Run sqlmap against a single target URL:
+
+`python sqlmap.py -u {{"http://www.target.com/vuln.php?id=1"}}`
+
+- Send data in a POST request (`--data` implies POST request):
+
+`python sqlmap.py -u {{"http://www.target.com/vuln.php" --data={{"id=1"}}`
+
+- Change default parameter delimiter (&):
```
Comment (agnivade) [inline pages/common/sqlmap.md]: "Change the parameter delimiter (& is the default):"

**Comment by hbostann** [comment]
@agnivade That's perfectly fine.

**Comment by benboeck** [comment]
Hello,

you should definitely include `-r` as it is the easiest way to get sqlmap to work with all kinds of cookies and auth headers.
Proposal:

- Use raw request (such as GET/POST) including cookies, authentication and other headers:

`sqlmap -r {{raw_request_file}}`

For important use cases I would also add:
- `--dbs` ("list databases")
- `-D {{database}} -T {{table}} --dump` ("dump contents of specific database table")

For advanced stuff (and as a teaser to read the sqlmap usage page ;)  `--os-shell` might be interesting.

**Comment by agnivade** [comment]
ping @sbrl @pxgamer @mfrw

**Pull request merged**